src/components/CowCard/Subheader/Subheader.js

```diff
@@ -150,7 +150,7 @@ const Subheader = (
               placement: 'top',
               title: (
                 <Typography>
-                  Check this box to put {cow.name} into a {huggingMachine.name}
+                  Check this box to put {cow.name} into a {huggingMachine.name} 
```

Comment:
I think tweaking this would ensure that the change doesn't get wiped out by automatic whitespace-removal tools:
```suggestion
                  Check this box to put {cow.name} into a {huggingMachine.name}{' '}
```